Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/ProgressIndicators/ProgressIndicatorOrbsRotator.cs

```diff
@@ -0,0 +1,207 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Threading.Tasks;
+using TMPro;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.SceneTransitions
+{
+    /// <summary>
+    /// This class manages the 'rotating circle of dots' effect
+    /// that is used as a Progress Indicator effect.
+    /// </summary>
+    public class ProgressIndicatorOrbsRotator : MonoBehaviour, IProgressIndicator
+    {
+        public Transform MainTransform { get { return transform; } }
+        public ProgressIndicatorState State { get { return state; } }
+        public float Progress { set { progress = value; } }
+        public string Message { set { messageText.text = value; } }
+
+        [SerializeField]
+        private GameObject[] orbs = null;
+        
+        [SerializeField]
+        private TextMeshPro messageText = null;
+
+        [SerializeField]
+        private ProgressIndicatorState state = ProgressIndicatorState.Closed;
+        
+        public float RotationSpeedRawDegrees;
```

Comment:
Nit: This is a bit inconsistent with the rest of the code that we have, which has private serialized fields, and the public setters and getters.

Why? I don't know, it doesn't necessarily make sense to me, but I'd still like for us to be consistent and stick to a given style without our codebase before consciously moving our entire coding style over.

@davidkline-ms please correct me if I'm wrong here.

Reply:
This is due to this being ported from a legacy control. I'll make these fields consistent.